### PR TITLE
Added logrotate configuration

### PR DESCRIPTION
**Pull request by thomaska**
Although Elasticsearch seems to be supporting log rotation (fun fact the support for this feature [was raised](https://github.com/elastic/elasticsearch/issues/7927) by @philmcmahon ) through configuring log4j, logrotate seems a more reliable solution. The end configuration is a mix of [modtools v1](https://github.com/guardian/discussion-modtools/blob/master/discussion/configs/logrotate/discussion-modtools.conf) and online examples ([here](http://stackoverflow.com/questions/28696069/elasticsearch-how-to-configure-logging-yml) and [here](http://serverfault.com/questions/656274/what-is-the-purpose-of-elasticsearch-logs-how-to-manage-them)).
In short it sets a limit in the number of the log files to 20 and max size to 50M each, thus a total of 1G.

### CODE REVIEW

**Comment by nicl** [comment]
@thomaska looks good but you'll need to rebase or merge in my changes. Let's also coordinate on the deploy to save us both doing this as it takes ages! :(

( 👍 )
